.github/workflows/build.yml

```diff
@@ -19,6 +19,7 @@ jobs:
             os: ubuntu-22.04
             cmake_args: >-
               -DQT6=ON
+              -DQML=OFF
```

Comment:
This is because with have two Ubuntu builds: one non-QML and one QML so I thought it would be great to keep the two builds so we can make sure the legacy, non-QML still builds fine. Alternatively, I could remove the second Ubuntu build and keep a QML only.

Reply:
Let me know if that change sounds sensible or if you would prefer me to remove the job.

Reply:
In my opinion, we should remove the job and build QML always on all platforms.
But we need instead test cases which ensure that both, QML and non-QML main-windows come up after start on each operating system (this is of cause, out of scope of this PR).

Reply:
Yes, that makes sense. Currently, we are unable to run QML on a stock Windows (and MacOS?), as it looks like the applocal would need additional work to include all the required dependencies

Reply:
@daschuer What's your opinion here?

Reply:
Yes, let's get rid of the dedicated QML build and save some CO2.